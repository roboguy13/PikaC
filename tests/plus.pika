%generate plus

plus : Int -> Int -> Int;
plus x y := x + y;

%test "plus1" Int: plus 2 2;