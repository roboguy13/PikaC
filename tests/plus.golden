void plus(loc i1, loc i, loc i2);
void plus(loc i1, loc i, loc i2) {
 WRITE_LOC(i2, 0, (long)i1 + (long)i);
}